grammar/q_util.mlp

```diff
@@ -94,27 +94,33 @@ let coincide s pat off =
   done;
   !break
 
+let check_separator sep =
+  if sep <> "" then failwith "Separator is only for lists."
+
 let rec parse_user_entry s sep =
   let l = String.length s in
   if l > 8 && coincide s "ne_" 0 && coincide s "_list" (l - 5) then
     let entry = parse_user_entry (String.sub s 3 (l-8)) "" in
-    Ulist1 entry
+    if sep <> "" then Ulist1sep (entry,sep) else Ulist1 entry
```

Comment:
I agree that the distrinction `list_sep` / `list` is somewhat irrelevant, be we should stick to uniformity there. Either we remove `list_sep` by merging it with `list`, or we fail like other entries when `list` is given a separator.

Reply:
OK, your position is also ok to me. Unless other voices raise, I shall preserve the original rationale.